### DISCUSSION TITLE
Dịch operation với tính toán tensor

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#123] [Terminology] [answered]
Từ `operation` với context là các phép tính tensor được sử dụng rất nhiều trong chương 12. Hiện tại mình đang dịch là `toán tử`. 

cc: @mlbvn/handson-ml 

__Context:__

>TensorFlow’s API revolves around tensors, which flow from operation to operation—hence the name TensorFlow.

>At the lowest level, each TensorFlow operation (op for short) is implemented usinghighly efficient C++ code. 2 Many operations have multiple implementations called kernels: each kernel is dedicated to a specific device type, such as CPUs, GPUs, or even TPUs (tensor processing units)

**Comment by minhduc0711** [comment]
Hình như đúng ra thì `operator` dịch là `toán tử`, còn `operation` là phép toán. Nhưng mà mình không phân biệt được hai từ này trong context lập trình nên chắc dịch vậy cũng ok. :sweat:

**Reply by quangnhat185** [reply]
Vậy tạm thời mình dịch `operation` là `toán tử`, sau này có thể rà soát lại nếu muốn sửa

**Comment by minhduc0711** [comment, marked as answer]
Chốt "operation" trong TensorFlow là "toán tử".
